### ISSUE TITLE
istio service dashboard not showing a lot of data -> N/A

### ISSUE THREAD

**Issue by sevaho** [istio/istio#7773]
**Describe the bug**

Having an nginx pod running with sidecar injected, but the grafana dashboard does not show a lot of data. Only client/server request duration and client request volume.

**Expected behavior**

I think it should show more datapoints

**Steps to reproduce the bug**
Steps to reproduce the behavior.

**Version**
What version of istio and Kubernetes are you using? Use `istioctl version` and `kubectl version`

istio 1.0
kubernetes 1.9.6

**Is Istio Auth enabled or not?**
Did you install the stable istio.yaml, istio-auth.yaml.... or if using the Helm chart please provide full command line input.

I followed this:
https://istio.io/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template

But I enabled grafana.

**Environment**
Which environment, cloud vendor, OS, etc are you using?

GKE

**Cluster state**
If you're running on Kubernetes, consider [following the
instructions](http://istio.io/help/bugs/#generating-a-cluster-state-archive)
to generate "istio-dump.tar.gz", then attach it here by dragging and dropping
the file onto this issue.

images:

https://imgur.com/a/rKy3PIc
https://imgur.com/a2x3f8z

**Comment by douglas-reid** [comment]
Are you certain that your service was receiving traffic?  Was your workload kube-injected?  Did you see any traffic to the underlying workloads (in the workload dashboard)?

Have you consulted the operations guide for telemetry: https://istio.io/help/ops/telemetry/ ?

**Comment by pravi1991** [comment]
Hi,
Even I am using the same setup as given above and I see the same results.
Some of my dashboards are empty.

I digged deeper using prometheus and ran these empty graph's prometheus query and I was able to find that the reporter label for **istio_requests_total** was showing as "destination" for bookinfo app and "source" for my custom app. The only difference between my app and bookinfo is the image used and the names and labels given. But still the label "destination_app" for bookinfo app shows as "productpage-v1" and for my app it is "istio-telemetry".
